modules/tree/src/main/ParseImport.scala

```diff
@@ -48,7 +48,9 @@ val parseImport: PgnStr => Either[ErrorStr, ImportResult] = pgn =>
             .map(Fen.write)
 
           val status = parsed.tags(_.Termination).map(_.toLowerCase) match
-            case Some("normal")                           => game.situation.status | Status.Resign
+            case Some("normal") =>
+              game.situation.status | (if parsed.tags(_.Result).getOrElse("") === "1/2-1/2" then Status.Draw
```

Comment:
`.getOrElse("") === "1/2-1/2"` == `.contains("1/2-1/2")`

Reply:
also this is not the place to hardcode the draw notation, scalachess has parsers